### ISSUE TITLE
[QUESTION] [iOS] Possible to load HTTP sites 

### ISSUE THREAD

**Issue by Flying--Dutchman** [SKLn-Rad/Xam.Plugin.Webview#32]
Hello, 

is there any possibility to load HTTP sites within iOS? 
Maybe using the SFSafariViewController?

**Comment by SKLn-Rad** [comment]
Its in the documentation!
You need to disable ATS in your plist.info

<key>NSAppTransportSecurity</key>
<dict>
<key>NSAllowsArbitraryLoads</key><true/>
</dict>

**Comment by Flying--Dutchman** [comment]
Strange, I already inserted that in my info.plist. Still, http sites won't load. 
https sites work just fine

![image](https://user-images.githubusercontent.com/9158539/28833752-9853db0c-76e0-11e7-921a-3965f90708aa.png)

**Comment by SKLn-Rad** [comment]
What error do you get back? Do you have any examples of sites which don't work?
I will give it a go when I get home

**Comment by Flying--Dutchman** [comment]
I'm losing all hope with Xamarin at this point. The entire day building new iOS packages with no success.
I just killed the build proccess on the Mac, cleaned the solution, did a rebuild (3 times) got errors when I tried to install it on the iPad (error MT1001: Could not find an application at the specified directory).

Clicked on start and for some stupid reason everything works fine now.... Frustrating

Guess this topic can be closed. Thank you for your support 👍
